lib/cocina/models/mapping/to_mods/form.rb

```diff
@@ -183,7 +183,7 @@ def write_physical_description_form_values(form_values, form: nil)
               attributes = {
                 unit: unit_for(form_value)
               }.tap do |attrs|
-                if PHYSICAL_DESCRIPTION_TAG.fetch(form_type) == :form && form_type != 'form'
+                if PHYSICAL_DESCRIPTION_TAG.fetch(form_type, nil) == :form && form_type != 'form'
```

Comment:
this isn't fixing the problem.  I will need to invoke Arcadia.

Reply:
(i did invoke Arcadia and we came up with a better solution in dor_indexing_app)